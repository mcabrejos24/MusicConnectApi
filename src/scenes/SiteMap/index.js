export default function SiteMap() {
    
    return (
        <div className="content site-map-page">
            Sitemap
        </div>
    );
}
