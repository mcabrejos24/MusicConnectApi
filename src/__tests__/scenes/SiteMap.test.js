import React from "react";
import { shallow } from "enzyme";
import SiteMap from '../../scenes/SiteMap';
import toJson from "enzyme-to-json";

describe("renders SiteMap", () => {
    let page;
    beforeEach(() => {
        page = shallow(<SiteMap />);
    });
    it("SiteMap loads", () => {
        page;
    });
    it("props class name matches", () => {
        expect(page.props().className).toEqual("content site-map-page");
    });
    it("matches snapshot", () => {
        expect(toJson(page)).toMatchSnapshot();
    });
});
