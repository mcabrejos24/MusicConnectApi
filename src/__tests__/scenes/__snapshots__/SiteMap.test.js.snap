// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SiteMap matches snapshot 1`] = `
<div
  className="content site-map-page"
>
  Sitemap
</div>
`;
